HeterogeneousCore/CUDAUtilities/interface/radixSort.h

```diff
@@ -0,0 +1,266 @@
+#ifndef HeterogeneousCoreCUDAUtilities_radixSort_H
+#define HeterogeneousCoreCUDAUtilities_radixSort_H
+
+#include <cstdint>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/cuda_assert.h"
+
+template <typename T>
+__device__ inline void dummyReorder(T const* a, uint16_t* ind, uint16_t* ind2, uint32_t size) {}
```

Comment:
The header consistency check complains
```
src/HeterogeneousCore/CUDAUtilities/interface/radixSort.h:9:1: error: '__device__' does not name a type; did you mean '__divtc3'?
 __device__ inline void dummyReorder(T const* a, uint16_t* ind, uint16_t* ind2, uint32_t size) {}
```
Should we protect CUDA-only parts in all headers with `ifdef __CUDACC__` then?

@smuzaffar @mrodozov

Reply:
What is the header consistency check,  "include what you use" ?
Maybe it will be happy once we add rules for building CUDA code with clang ?